bin/lisky.js

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env node
```

Comment:
Is `bin` the right location for this file? See https://docs.npmjs.com/files/package.json#directoriesbin

Reply:
That was my first thought, but I found projects where they put source file in the bin and build it there. (but I forgot which project it was)
At the same time, putting the file in other place will be okay too, but it's hard to say where is the correct place to put.